components/features/dashboard/home/InvitationLinksList.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+
+import axios from "@/lib/axiosInstance";
+import { AxiosError } from "axios";
+import { useQuery } from "@tanstack/react-query";
+import { getCookie } from "cookies-next";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { Button } from "@/components/ui/button";
+import { Copy } from "lucide-react";
+import { capitalizeFirstLetter } from "@/lib/utils";
+import DeleteInvitationLink from "@/components/features/dashboard/home/DeleteInvitationLink";
+import { useState } from "react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+
+type Props = {
+  schoolId: string;
+};
+
+async function getSchoolInvitations(schoolId: string) {
+  const token = await getCookie("token");
+  try {
+    const res = await axios.get(`school/${schoolId}/invitation`, {
+      headers: { Authorization: token },
+    });
+    return res.data;
+  } catch (err) {
+    const error = err as AxiosError;
+    console.error(
+      (error.response && error.response.data) || "Unexpected error occur",
+    );
+  }
+}
+
+type SchoolInvitationType = {
+  id: number;
+  token: string;
+  role: "ADMIN" | "TEACHER" | "STUDENT";
+  createdAt: string;
+  updatedAt: string;
+};
+
+export default function InvitationLinksList({ schoolId }: Props) {
+  const host = window.location.host;
```

Comment:
Include the protocol.
```suggestion
  const host = (window.location.protocol + "//" + window.location.hostname);
```